docs/external_libraries.rst

```diff
@@ -0,0 +1,36 @@
+Extension libraries
+===================
+
+The Taichi programming language offers a minimal and generic built-in standard library. Extra domain-specific functionalities are provided via **extension libraries**:
+
+
+Taichi GLSL
+-----------
+
+`Taichi GLSL <https://github.com/taichi-dev/taichi_glsl>`_ is an extension
+library of Taichi, aiming at providing useful helper functions including:
+
+1. Handy scalar functions like ``clamp``, ``smoothstep``, ``mix``, ``round``.
+2. GLSL-alike vector functions like ``normalize``, ``distance``, ``reflect``.
+3. Well-behaved random generators including ``randUnit2D``, ``randNDRange``.
```

Comment:
Would "random number generators" be better here? :D

Reply:
The rand unit in Taichi GLSL also supports random vectors, not sure if number will imply scalars..